.github/ISSUE_TEMPLATE/bug_new.yml

```diff
@@ -11,9 +11,8 @@ body:
     id: what-happened
```

Comment:
Aligned the bug form to the Markdown one, and renamed it to be able to test it as otherwise it wasn't showing. If it works fine, we can remove the markdown